### ISSUE TITLE
noiice-dev-nuxt fails to serve the app

### ISSUE THREAD

**Issue by airpaio** [DylanAllen/noiiice#27]
I was trying a fresh deploy of noiiice from the latest master branch commit 41f2354, and the deployment seems to go through without any errors. However, trying to visit dev.domain.com I get a 502 error. 

The Lambda function noiice-dev-nuxt logs the following error:

```
{
  "errorType": "Runtime.ImportModuleError",
  "errorMessage": "Error: Cannot find module 'serverless-http'\nRequire stack:\n- /var/task/index.js\n- /var/runtime/UserFunction.js\n- /var/runtime/index.js",
  "trace": [
    "Runtime.ImportModuleError: Error: Cannot find module 'serverless-http'",
    "Require stack:",
    "- /var/task/index.js",
    "- /var/runtime/UserFunction.js",
    "- /var/runtime/index.js",
    "    at _loadUserApp (/var/runtime/UserFunction.js:100:13)",
    "    at Object.module.exports.load (/var/runtime/UserFunction.js:140:17)",
    "    at Object.<anonymous> (/var/runtime/index.js:43:30)",
    "    at Module._compile (internal/modules/cjs/loader.js:1137:30)",
    "    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1157:10)",
    "    at Module.load (internal/modules/cjs/loader.js:985:32)",
    "    at Function.Module._load (internal/modules/cjs/loader.js:878:14)",
    "    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:71:12)",
    "    at internal/main/run_main_module.js:17:47"
  ]
}
```

When I inspect the Lambda Layer in this function, it only has the files `package.json` and `package-lock.json`. The deploy/build process does build the `node_modules` directory in `layer/nodejs/` but it's not being uploaded in the Lambda Layer.

**Comment by DylanAllen** [comment]
This is a bug, but if you deploy the full app again it should upload the node modules for the layer.

The problem is that the script to run `npm ci` on the layer happens after the deployment finishes. Which is too late the first time. The app is redeployed after the build, but the layer is not. I don't think serverless has a way to deploy just a layer, so I need to re-work the scripts to run the install before deployment.

**Comment by DylanAllen** [comment]
Resolved with commit  c07395c
